specification/enrich/execute_policy/ExecuteEnrichPolicyRequest.ts

```diff
@@ -21,15 +21,34 @@ import { RequestBase } from '@_types/Base'
 import { Name } from '@_types/common'
 
 /**
+ * Creates the enrich index for an existing enrich policy.
+ * The enrich index contains documents from the policy’s source indices. Enrich
+ * indices always begin with `.enrich-*`, are read-only, and are force merged.
+ * Enrich indices should be used by the enrich processor only. Avoid using
+ * enrich indices for other purposes.
+ * Once created, you cannot update or index documents to an enrich index.
+ * Instead, update your source indices and execute the enrich policy again. This
+ * creates a new enrich index from your updated source indices. The previous
+ * enrich index will deleted with a delayed maintenance job. By default this is
+ * done every 15 minutes.
+ * Because this API request performs several operations, it may take a while to
+ * return a response.
```

Comment:
Maybe this is too much information for reference docs? I'd stick to just _"Creates the enrich index for an existing enrich policy."_ here, and maybe add a `@doc_id` for more information.